.gitignore

```diff
@@ -17,3 +17,5 @@ node_modules
 npm-debug.log
 yarn-error.log
 .betterer.cache
+.yalc
+yalc.lock
```

Comment:
You can add those to your global gitignore file.

Reply:
lets remove this before merge

Reply:
https://medium.com/@peter_graham/how-to-create-a-local-gitignore-1b19f083492b

Reply:
https://github.com/wireapp/wire-webapp/pull/13281/commits/60ad10081e1a215c974fc53d563b90c31f776cbe